etc/flux.service.in

```diff
@@ -4,7 +4,7 @@ Wants=munge.service
 
 [Service]
 Type=notify
-NotifyAccess=main
+NotifyAccess=all
```

Comment:
Is there a missing "to" in the commit message "Change _to_ NotifyAccess=all (from =main)..."?